### ISSUE TITLE
Update Core Member list on "About"

### ISSUE THREAD

**Issue by mgottholsen** [civicdata/civicdata.github.io#43]
```
*Current Core Members*
@Cody Mcpheron
@Aaron Marks
@Bethany
@beckysteele
@bret
@chrisharrell
@eric_bickel
@502eire
@jessetylermullins
@leigh
@margeaux
@mikelcu
@pat
@rkahne
@shelleyhicklin
@thebouv
@zbouvier
```

**Comment by schnuerle** [comment]
What is the definition of core member? Is it on the site or in CoC? Is not maybe add it to the site or link to it from the site.

**Comment by jessetylermullins** [comment]
>What is the definition of core member?

@schnuerle We're in the process of putting docs that include the answer to that, and more, together. They'll be hosted in the repo and linked to from the site.

**Comment by pat** [comment]
Might want to double-check those usernames, as I'm quite sure I'm not a member of any sort, core or otherwise :)

**Comment by mgottholsen** [comment]
Ah but you COULD be @pat. Civic Data Alliance, Melbourne branch?

My apologies. That's what I get for copying and pasting out of Slack.

**Comment by pat** [comment]
No worries! If only I could reserve @pat in all the places ;)

**Comment by jessetylermullins** [comment]
Addressed in PR #60

**Comment by jessetylermullins** [comment]
#60 was merged, closing this issue
